CSharp/DevVmPowershell/DevVmPsModules/Cmdlets/ShowInstallerAndInstanceRelativityVersionAreEqualModule.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+using Helpers.Implementations;
+using Helpers.Interfaces;
+using Relativity.Services.Document;
+
+namespace DevVmPsModules.Cmdlets
+{
+	[Cmdlet(VerbsCommon.Show, "InstallerAndInstanceRelativityVersionAreEqual")]
```

Comment:
_Confirm_ seems like a correct verb instead of _Show_. Please refer to the approved verb list here - https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7